staging/src/k8s.io/code-generator/pkg/util/build.go

```diff
@@ -50,3 +53,24 @@ func hasSubdir(root, dir string) (rel string, ok bool) {
 	// cut off root
 	return filepath.ToSlash(dir[len(root):]), true
 }
+
+// BoilerplatePath uses the boilerplate in k8s.io/code-generator.
+// If it does not exist, it uses the boilerplate in k8s.io/kubernetes.
+func BoilerplatePath() string {
+	boilerPlatePath := "k8s.io/kubernetes/vendor/k8s.io/code-generator/hack/boilerplate.go.txt"
```

Comment:
can the upper case go away now?

Reply:
do you mean making it non-exported like `boilerplatePath` or changing the name like `BoilerpatePath`?

The former cannot happen as this is used from the util package like in: https://github.com/kubernetes/kubernetes/pull/57656/files#diff-9463026ba452a5431275eb6d46eabd79R38

Reply:
sorry, was misreading. The kube path is gone already :+1:

Reply:
The old file in hack/ still exists? Can you remove that as well?

Reply:
> The old file in hack/ still exists? Can you remove that as well?

It is already removed.

https://github.com/kubernetes/kubernetes/pull/57656/files#diff-6f2402d716134738fb1648fa419ab9b3

Github makes it hard to see/review though. :/

Reply:
:+1: